src/sentry/tsdb/base.py

```diff
@@ -12,6 +15,12 @@
 ONE_DAY = ONE_HOUR * 24
 
 
+@dataclasses.dataclass
+class IncrMultiOptions:
```

Comment:
is this right? I don't see anything constructing this

Reply:
It's used as a type on `incr_multi` parameter

Reply:
yeah I see that part -- but this is a concrete class so something else would need to be constructing this and passing it in for that to ever be possible?